tests/cli/base.py

```diff
@@ -74,3 +81,160 @@ def command_test(self, arg, kwarg=None, kwarg2=False):
                 pass
         argv = 'mage.py test'
         self.assertRaises(SystemExit, lambda: manage(dict(test=TestCommand()), argv.split()))
+
+    def test_digest_not_found(self):
+        class TestCommand(Cli):
+            def command_test(self, kwarg=None, kwarg2=False):
+                pass
+        argv = 'mage.py test1:test --kwarg --kwarg2'
+
+        out = StringIO()
+        with patch.object(sys, 'stdout', out):
+            with self.assertRaises(SystemExit):
+                manage(dict(test=TestCommand()), argv.split())
+            self.assertEqual('Commands:\ntest\n', out.getvalue())
+
+    def test_no_command_provided(self):
+        class TestCommand(Cli):
+            def command_test(self, kwarg=None, kwarg2=False):
+                pass
+        argv = 'mage.py'
+
+        out = StringIO()
+        with patch.object(sys, 'stdout', out):
+            with self.assertRaises(SystemExit):
+                manage(dict(test=TestCommand()), argv.split())
+            self.assertEqual('Commands:\ntest\n', out.getvalue())
+
+    def test_test_command_not_found(self):
+        class TestCommand(Cli):
+            def command_test(self, kwarg=None, kwarg2=False):
+                pass
+        argv = 'mage.py test:test1 --kwarg --kwarg2'
+
+        out = StringIO()
+        with patch.object(sys, 'stdout', out):
+            with self.assertRaises(SystemExit):
+                manage(dict(test=TestCommand()), argv.split())
+            self.assertEqual('test\ntest\n\t./mage.py test:test [kwarg] [kwarg2]\n',
```

Comment:
This is not error message, this is help message, and, as I judging by Cli:description, adding "./" is desired behaviour.